crates/topos/src/components/node/mod.rs

```diff
@@ -148,7 +148,7 @@ pub(crate) async fn handle_command(
                      edge arguments: {:?}",
                     genesis.path.display(),
                     data_dir.display(),
-                    config.edge.as_ref().unwrap().args
+                    config.edge.as_ref().expect("valid edge configuration").args
```

Comment:
I don't see where the `edge` key is actually validated so that we can be sure that it's there at this point.

Reply:
If there is no edge configuration it does not make sense to startup node

Reply:
Updated: fit to exit gracefully if edge config is missing

Reply:
Lovely, thank you for humoring my nitpicks. :)